perfkitbenchmarker/linux_packages/mongodb_server.py

```diff
@@ -18,6 +18,7 @@
 
 def YumInstall(vm):
   """Installs the mongodb package on the VM."""
+  vm.RemoteCommand('sudo setenforce 0')
```

Comment:
Related links;
- https://www.centos.org/docs/5/html/5.2/Deployment_Guide/sec-sel-enable-disable-enforcement.html
- What is SELinux - http://www.linux.com/learn/answers/view/394-what-is-selinux-and-how-does-it-work-

My Question... Can you turn this back on after the install and does MongoDB continue to run properly?